modules/container.nix

```diff
@@ -0,0 +1,131 @@
+{
+  context,
+  system,
+  config,
+  lib,
+  ...
+}: let
+  floxpkgs = context.inputs.flox-floxpkgs;
+in {
+  options.container = with lib; {
+    name = mkOption {
+      description = mdDoc ''The name of the resulting image.'';
+      type = types.str;
+      default = "floxEnv";
```

Comment:
@ysndr what's the best way to get the name of a floxEnv? Is it somewhere in context? I.e. if this floxEnv is in pkgs/name/flox.nix I want name.

Reply:
@mkenigs are you looking for "namespace"? https://github.com/flox/floxpkgs/blob/master/plugins/floxEnvs.nix#L14

Reply:
its not in the context, but you have access to it in the plugin and can pass it into lib.mkFloxEnv -> the module

Reply:
Pushed a commit to thread namespace all the way through. Alternatively I could have called `builtins.elemAt namespace (builtins.length namespace - 1)` in the plugin and threaded name through - not sure which is better